wca-regulations.md

```diff
@@ -328,6 +328,18 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 11f) Decisions about an incident may be supported with video or photographic analysis, at the discretion of the WCA Delegate.
 - 11g) The WCA Delegate must ensure that copies of the Regulations and Guidelines are available (e.g. printed, digital, or accessible via internet) to officials and competitors for consultation on any incidents.
 - 11h) A WCA Delegate may require competitors in serious violation of [Regulation 2k3](regulations:regulation:2k3) to leave the competition venue, taking into account the seriousness of the situation and the best possible course of action. If the competitor refuses to do so, they may face disciplinary action in the WCA.
+- 11i) If it is found that a competitor's puzzle was not scrambled correctly after they have completed their solve, the following regulations apply:
+    - 11i1) In the following cases, an extra attempt must be granted to replace the affected attempt. If no extra attempt is granted, the result will be considered DNS:
+        - 11i1a) The individual result, or the average or mean the result is part of, is a regional record or a personal record in the top 50 results of the world ranking.
```

Comment:
This wording is very confusing. My recommendation is to split single and average into 2 seperate regulations:
- 11i1a) The individual result is a regional record or a personal record in the top 50 results of the world rankings.
- 11i1b) The average or mean the result is a part of is a regional record or a personal record in the top 50 results of the world rankings.

It's an extra regulation with a lot of overlap, however it is significantly easier to comprehend the regulations.

Reply:
Adding onto this, although I am not as concerned about the wording as Jacob is, what is the base reference for the Top 50 rankings?

Consider the following example:
* The current WR50 Clock average is 3.79 seconds (held by several people, but the top result is WR50).
* The current WR49 Clock average is 3.77 seconds.
* During the first group of Clock at a competition, Competitor A achieves a 3.78 second average, which is now WR50.
* During the next group at the same competition, Competitor B achieves a 3.77 second average (tying WR49, so Competitor A is now WR51).
* After the round (but still during the competition), Competitor A notices a misscramble from their earlier 3.78 average.

At the time of the average, it placed WR50. However, it is now WR51. Would an extra attempt be required to be given to Competitor A since their result would have been WR50 at the time of the average? Or should the misscramble be allowed to stand since the result is now out of the top 50?

Reply:
I like Jacob's wording better than the current wording. I would even change the proposed 11i1b) to something like this:

11i1b) The result is part of an average or mean that is a regional record or a personal record in the top 50 results of the world rankings.

Reply:
Thanks for bringing up some of these things!

> What if a result is determined to not fall under the criteria for an extra at the competition due to a recent result, and the recent result later gets disqualified (eg. also due to a misscramble)?
> Adding onto this, although I am not as concerned about the wording as Jacob is, what is the base reference for the Top 50 rankings?

I think these edge cases can be generally addressed by using "at the time of achievement". These should be very rare cases hopefully. The Delegate can always provide a provisional extra if it is something really close to top 50.

> What happens when an average contains two misscrambles? Should only the faster be disqualified if disqualifying it prevents the result from being top 50/NR?

I would think both. If one average (for a high level competitor no less!) includes multiple misscrambles though, that is indicative of a larger problem. 

I also do like Jacob's wording right now